Make fmt target modify files

leighmcculloch opened the pull request:
### What
Make the `make fmt` target of projects modify files, instead of erroring and displaying a diff.

### Why
The `make fmt` target used to update unformatted files in this repository, as it does in other repositories. It appears that recently it was changed to check, error, and display a diff, which is less convenient. I think this change was so that the `fmt` GitHub Action job could call a single command to run a check. We can get the same behavior in the GitHub Action by using rustfmt directly.

<a href="https://gitpod.io/#https://github.com/stellar/soroban-examples/pull/247"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>

anupsdf commented:
I had the same `file `**/*.rs` does not exist` error with rustfmt. You could try $(wildcard **/*.rs) as per this [make tutorial](https://makefiletutorial.com/).

leighmcculloch commented:
@anupsdf I've updated the PR now and it passes. I'm avoiding using the makefile target, because I changed the makefile target not to error and display a diff, but instead fix the formatting.

anupsdf reviewed (approved):
LGTM
